simplefs/simplefs.go

```diff
@@ -508,12 +527,7 @@ func (k *SimpleFS) SimpleFSWrite(ctx context.Context, arg keybase1.SimpleFSWrite
 // SimpleFSRemove - Remove file or directory from filesystem
 func (k *SimpleFS) SimpleFSRemove(ctx context.Context,
 	arg keybase1.SimpleFSRemoveArg) error {
-	go k.simpleFSRemove(context.Background(), arg)
-	return nil
-}
-func (k *SimpleFS) simpleFSRemove(ctx context.Context,
-	arg keybase1.SimpleFSRemoveArg) (err error) {
-	return k.startAsync(arg.OpID, keybase1.NewOpDescriptionWithRemove(
+	return k.startAsync(ctx, arg.OpID, keybase1.NewOpDescriptionWithRemove(
```

Comment:
The extra `go` wasn't needed, since this was already an async operation.  I assume it was leftover from some historic code organization.